queries/dune_v1/period_slippage.sql

```diff
@@ -8,14 +8,21 @@ filtered_trades as (
            solver_name,
            solver_address,
            trader                                                as trader_in,
-           receiver                                              as trader_out,
+           -- Null Receiver: https://dune.com/queries/1729130
+           case
+                when receiver = '\x0000000000000000000000000000000000000000'
+                then trader
+            else receiver end                                    as trader_out,
            sell_token_address                                    as "sellToken",
            buy_token_address                                     as "buyToken",
-           atoms_sold                                            as "sellAmount",
+           atoms_sold - coalesce(surplus_fee, 0)                 as "sellAmount",
            atoms_bought                                          as "buyAmount",
            '\x9008D19f58AAbD9eD0D60971565AA8510560ab41' :: bytea as contract_address
     from gnosis_protocol_v2."trades" t
              join gnosis_protocol_v2."batches" b on t.tx_hash = b.tx_hash
+    left outer join dune_user_generated.cow_order_rewards_barn f
```

Comment:
notice this queries the "barn" table now -- which will only be updated at 6PM (+/- 1 hour) today. If you plan on executing this yourself, you should wait.

Reply:
I think you are misunderstanding the difference between these tables and maybe I should not have used these names. These are our staging and production version of deployment. Both tables contain everything. The prod table does not even exist because this service is only deployed on staging. 

I will fix these names so there is no more confusion.